hazelcast/src/main/java/com/hazelcast/concurrent/lock/operations/UnlockOperation.java

```diff
@@ -65,19 +66,44 @@ public void run() throws Exception {
     }
 
     protected final void unlock() {
+        ILogger logger = getLogger();
+
+        if (logger.isFinestEnabled()) {
+            logger.finest("Releasing lock " + namespace.getObjectName());
+        }
+
         LockStoreImpl lockStore = getLockStore();
         boolean unlocked = lockStore.unlock(key, getCallerUuid(), threadId, getReferenceCallId());
         response = unlocked;
         if (!unlocked) {
             // we can not check for retry here, hence just throw the exception
             String ownerInfo = lockStore.getOwnerInfo(key);
             throw new IllegalMonitorStateException("Current thread is not owner of the lock! -> " + ownerInfo);
+        } else if (logger.isFinestEnabled()) {
+            logger.finest("Released lock " + namespace.getObjectName());
         }
     }
 
     protected final void forceUnlock() {
+        ILogger logger = getLogger();
+
+        if (logger.isFinestEnabled()) {
+            logger.finest("Forcibly releasing lock " + namespace.getObjectName()
+                    + " by " + getCallerAddress() + " - " + getCallerUuid());
+        }
+
         LockStoreImpl lockStore = getLockStore();
-        response = lockStore.forceUnlock(key);
+        boolean unlocked = lockStore.forceUnlock(key);
+        this.response = unlocked;
+
+        if (logger.isFinestEnabled()) {
+            if (unlocked) {
+                logger.finest("Released lock " + namespace.getObjectName());
+            } else {
+                logger.finest("Could not release lock " + namespace.getObjectName()
```

Comment:
Could you double check this? It seems the `false` is returned if no such lock exist.

The `LockStoreImpl` contains:
```
    public boolean forceUnlock(Data key) {
        LockResourceImpl lock = locks.get(key);
        if (lock == null) {
            return false;
        }
...
```